foundation_cms/legacy_apps/templates/pages/base.html

```diff
@@ -112,7 +112,7 @@
 
         {% block donate_banner %}
             {% if donate_banner %}
-                {% include "fragments/donate_banner.html" with banner=donate_banner %}
+                {% include "patterns/components/donate_banner.html" with banner=donate_banner %}
```

Comment:
This legacy template has been updated to use the **new** donate banner template. That way, legacy pages such as blog/ or campaigns/ that live under the redesign homepage show the correct donate banner if one is being used.